README.md

```diff
@@ -1,20 +1,37 @@
 <!-- markdownlint-disable MD034 -->
 # Container Storage Interface (CSI) driver for vSphere
 
-This repository provides tools and scripts for building and testing the vSphere CSI provider. This driver is in a stable `GA` state and is suitable for production use. Some of the features may be in the `beta` phase. Please refer [feature matrix](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-E59B13F5-6F49-4619-9877-DF710C365A1E.html) for more details.  vSphere CSI driver requires vSphere 6.7 U3 or higher in order to operate.
+The vSphere CSI Driver is a Kubernetes plugin that allows persistent storage for containerized workloads running on vSphere infrastructure. It enables dynamic provisioning of storage volumes and provides features like snapshots, cloning, and dynamic expansion of volumes. The vSphere CSI Driver replaces the in-tree vSphere storage plugin and offers better scalability, flexibility, and compatibility with different Kubernetes versions.
+
+This driver is in a stable `GA` state and is suitable for production use. Some features may be in the `beta` phase. Please refer [feature matrix](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html#GUID-E59B13F5-6F49-4619-9877-DF710C365A1E) for more details.  
+
+The vSphere CSI Driver is supported on vSphere 6.7U3 and later versions.
+If you are using an earlier version of vSphere, you may need to upgrade to a supported version to use the vSphere CSI Driver.
+It's also important to note that the vSphere CSI Driver has different versions, and each version may have different requirements or limitations, so it's essential to check the [documentation](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html) for your specific version.
 
 The CSI driver, when used on Kubernetes, also requires the use of the out-of-tree vSphere Cloud Provider Interface [CPI](https://github.com/kubernetes/cloud-provider-vsphere).
```

Comment:
We removed the dependency on vSphere CPI, right?

Reply:
We are not using providerID set by vSpehre CPI anymore for node discovery but to perform cloud provider operations on the k8s cluser deployed on vSphere we do require CPI to be installed.

Reply:
clarified as below

> Kubernetes on vSphere requires the use of the [out-of-tree vSphere Cloud Provider Interface (CPI)](https://github.com/kubernetes/cloud-provider-vsphere) to perform [cloud provider](https://github.com/kubernetes/cloud-provider) operations.
vSphere CSI Driver does not require vSphere CPI to be installed as prerequisite.